tests/bsim/bluetooth/compile.nrf5340bsim_nrf5340_cpuapp.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright 2018 Oticon A/S
```

Comment:
Is this the correct copyright?

Reply:
Well.. the file started as a `cp` of `compile.sh`, so I tend to not change copyrights when I copy and modify lightly a file.
The whole file is so trivial it feels irrelevant to add an extra copyright line.

Reply:
Fair :)